src/aspire/config_default.yaml

```diff
@@ -5,6 +5,14 @@ common:
     # fft backend to use - one of pyfftw/scipy/cupy/mkl
     fft: pyfftw
 
+    # Set cache directory for ASPIRE example data.
+    # By default the cache location will be set by pooch.os_cache(),
+    # which sets cache based on operating system as follows:
+    # Mac: ~/Library/Caches/<AppName>
+    # Unix: ~/.cache/<AppName> or the value of the XDG_CACHE_HOME environment variable, if defined.
+    # Windows: C:\Users\<user>\AppData\Local\<AppAuthor>\<AppName>\Cache
+    cache_dir: ""
```

Comment:
Yes, this uses the default. Note that in `src/aspire/__init__.py` we use `pooch.os_cache` if this string is empty. Otherwise, we use a cache that has been set by the user.